docs/pages/cloud/faq.mdx

```diff
@@ -61,43 +62,101 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
+Individual customers may not be upgraded to the latest release, if they are running agents that are outside of the supported compatibility of the control plane.
 
-### Does your SOCII report include Teleport Cloud?
+## Does your SOCII report include Teleport Cloud?
 
 (!docs/pages/includes/soc2.mdx!)
 
-### Can a customer deploy multiple clusters in Teleport Cloud?
+## Can a customer deploy multiple clusters in Teleport Cloud?
 
-Not at this time.
+[Reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
-### Is FIPS mode an option?
+## Is FIPS mode an option?
 
 FIPS is not currently an option for Teleport Cloud clusters.
 
-### How do you store passwords?
+## How do you store passwords?
 
 Password hashes are generated using
 [Golang's bcrypt](https://pkg.go.dev/golang.org/x/crypto/bcrypt).
 
-### How does Teleport manage web certificates? Can I upload my own?
+## How does Teleport manage web certificates? Can I upload my own?
 
 Teleport uses [letsencrypt.org](https://letsencrypt.org/) to issue
 certificates for every customer. It is not possible to upload a custom
 certificate or use a custom domain name.
 
-### Do you encrypt data at rest?
+## Do you encrypt data at rest?
 
 Each deployment is using at-rest encryption using AWS DynamoDB and S3 at-rest encryption
 for customer data including session recordings and user records.
+
+## Can I get a list of IP addresses to configure a firewall?
+
+We don't currently offer a list of addresses for firewall configuration, but intend to start offering this in early 2022.
+
+## Can I configure Teleport Cloud to only allow connections from a whitelist of IP addresses?
+
+No. We believe mTLS with strong identity is the best standard available for authenticating and access clients.
+We are also investigating additional and new features that can be used as a defense in depth strategy for use with Teleport Cloud in 2022.
+
+## Is IPv6 Supported for connections to Teleport Cloud?
+
+We don't currently support IPv6 connections to Teleport Cloud. 
+
+## Can I change the domain name of my Cloud instance after it's been created?
+
+We're currently researching whether this can be done, so please contact support.
+
+## Can I retrieve diagnostics from my hosted cluster?
+
+We currently don't expose any metrics interfaces for a tenant.
+
+For our own metrics collection, we're rolling out mTLS, so that only authorized clients may collect or scrape metrics from the running instances. 
+So we would need to extend our certificate management to external clients.
+
+Additionally, we would need to allow all components of a tenant to be individually addressable and reachable from outside our network (so a scraper can scrape the individual targets). 
+This could allow those individual components to be selectively attacked, if an adversary is able to address requests to specific targets within our network.
```

Comment:
I think we can make this sentence a bit tighter, e.g., "This could allow an adversary to attack these components by addressing requests to specific targets within our network."